javascript/javascript3/week1/hscript.js

```diff
@@ -0,0 +1,114 @@
+
+/*--------Homework--------JSON-------*/
+// let  berry = {
+//   number: 8
+// };
+
+// let eat = {
+//   title: "berries containing zinc",
+//   elementIsNecessaryForLife: [{name: "avocados"}, {name: "pomegranates"}, {name: "blackberriess"}, {name: "berries"}, {name: "blackberries"}, {name: "raspberries"}, {name: "loganberries"}, {name: "dates"}],
+//   place: berry // element is necessary for life
+// };
+
+// berry.occupiedBy = eat; //Sources of zinc are avocados, pomegranates, berries (blackberries, raspberries, loganberries,dates)
+
+// alert( JSON.stringify(eat, function replacer(key, value) {
+//   console.log(`${key}: ${value}`);
+//   return (key == 'occupiedBy') ? undefined : value;
+// }));
+/*-----------API---------------*/
+const api_url = 'https://api.wheretheiss.at/v1/satellites/25544';
+async function getISS() {
+	const response = await fetch(api_url);
+	const data = await response.json();
+	const { latitude, longitude} = data;
+
+	document.getElementById('lat').textContent = latitude;
+	document.getElementById('lon').textContent = longitude;
+ }
+	getISS();
+
+/* answer =  API is a set of functions and rules that exist.
+Inside the application "Application" .API simple interface between the application
+and other elements, such as third-party software or
+equipment. API is a set of code functions (methods, properties, events and url addresses)
+   Which can be used in their applications to interact with web browser components
+user or third-party web services. In my example, the program has a written program for
+Definitions of catching and latitude available via URL address.
+An API is a set of existing functions and rules.
+   Inside the application "Application". API is a simple interface between an application
+   and other items such as third party software or
+   equipment. An API is a set of code functions (methods, properties, events, and URLs).
+   Which can be used in your applications to interact with web browser components.
+users or third party web services. In my example, the program has a written program for
+The user's longitude and latitude definitions are available via a URL.
+	*/
+
+	/*------------------------------------*/
+	api.openweathermap.org/data/2.5/weather?q=Ranum&appid=833893ea6ddd4c9bc2f142d8c1316d62
+
+	http://api.openweathermap.org/data/2.5/forecast?id=524901&appid={833893ea6ddd4c9bc2f142d8c1316d62y}
+	 https://api.openweathermap.org/data/2.5/weather?q=copenhagen&appid=833893ea6ddd4c9bc2f142d8c1316d62
+let weather = {
+	apiKey: "a1f94e5c6976e42c8f7d8149416f21bf",
```

Comment:
is this api key working? When I open the index.html page and click the search button to find the weather for a city, it doesn't work and it seems something is wrong with this. Maybe it expired?